Automattic/newspack-theme
# Update Facebook's SVG icon

### All Submissions:






### Changes proposed in this Pull Request:



This PR updates Facebook's SVG icon to its new one.

See #1081

### How to test the changes in this Pull Request:

1. Add a social menu with a facebook link
2. Check the front-end -- You should see the old FB logo
3. Switch to this branch
4. Refresh front-end -- you should see the new FB logo

### Other information:
